Certified overview laptops

carkod opened the pull request:
## Done

- Certified overview laptops. 

## QA

- Check https://ubuntu-com-9775.demos.haus for the new laptops card


## Issue / Card

Fixes https://github.com/canonical-web-and-design/certification.ubuntu.com/issues/195

webteam-app commented:
Demo starting at https://ubuntu-com-9775.demos.haus

bethcollins92 commented:
There are no numbers after the releases. Also the numbers seem to be exactly the same between laptops and desktops - double checking is it just a coincidence?
![image](https://user-images.githubusercontent.com/58959073/118674450-b238cb80-b7f1-11eb-8e20-cea2dc392a9c.png)

codersquid commented:
> Also the numbers seem to be exactly the same between laptops and desktops - double checking is it just a coincidence?

The numbers should not be the same. See https://certification.canonical.com/api/v1/certifiedmakes/?format=json

carkod commented:
> There are no numbers after the releases. Also the numbers seem to be exactly the same between laptops and desktops - double checking is it just a coincidence?
> ![image](https://user-images.githubusercontent.com/58959073/118674450-b238cb80-b7f1-11eb-8e20-cea2dc392a9c.png)

Fixed. Also removed the path, since it's composed in the template, and removed the `filters=True` from there

bethcollins92 commented:
QA LGTM - not sure if Sheila wants to check the python code?

codersquid commented on webapp/certification/views.py:
```diff
@@ -195,6 +195,10 @@ def certified_home():
     certified_releases = api.certified_releases(limit="0")["objects"]
     certified_makes = api.certified_makes(limit="0")["objects"]
 
+    # Desktop section
```
Update the comment to say Laptops.

codersquid reviewed (commented):
I have a very minor comment about a comment. see inline comment.

codersquid reviewed (approved):
lgtm
